src/data.ts

```diff
@@ -85,19 +97,7 @@ export interface FooterData {
   copyright: string;
 }
 
-export interface RootObject {
-  cart: Cart[];
-  navigation: Navigation;
-  menu: Menu[];
-  backButtonTitle: BackButtonTitle;
-  carousel: Carousel[];
-  carouselButtons: CarouselButtons;
-  about: About;
-  gallery: GalleryData;
-  footer: FooterData;
-}
```

Comment:
why was this removed?
having this type helps you when you want to check the underlaying data structure
currently when you hover over cart property (line 101) in `defaultData` object you'll see that it's `never` which is not the case in your app :P